package.json

```diff
@@ -2,7 +2,9 @@
   "name": "single-spa-angular",
   "version": "3.4.1",
   "description": "Helpers for building single-spa applications which use Angular 2",
-  "main": "lib/browser-lib/single-spa-angular.js",
+  "main": "lib/browser-lib/cjs/single-spa-angular.js",
+  "module": "lib/browser-lib/esm/single-spa-angular.js",
```

Comment:
Does angular differential loading rely on the `main` and `module` fields in the package.json to differentiate between es5/es2015?

Reply:
Depends on the compiler.

* View Engine relies on `fesm2015` and falls back to `module` when it doesn't exist (since we do not use ng-packagr we can just emit single ES2015 file and set `module` property)
* Ivy relies on `fesm2015_ivy_ngcc`

Reply:
And the _differential loading_ is only a mechanism that does:
* reads _browserslist_ file (if exists)
* runs `require('browserslist')(...)`
* determines what browsers have to be supported
* if there are all browsers that do not support _es2015_ then target _es5_ only
* if there are all browsers that do support _es2015_ (e.g. `last 2 Chrome versions`) then target _es2015_ only
* if there are browsers that support _es2015_ & _es5_ then target _es2015_ and additionally emit _es5_ files and add scripts to HTML with `nomodule` attribute

That's it.